src/email.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+export { printEmailErrorMessage, isEmail };
+
+const inputEmail = document.querySelector('#email');
+const emailErrorMessage = document.querySelector('.email-error-msg');
+
+function isEmail(value) {
+  let regExp =
+    /^[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_\.]?[0-9a-zA-Z])*\.[a-zA-Z]{2,3}$/i;
+  return regExp.test(value);
+}
+
+// 이메일 입력란이 비어있거나, 이메일 형식이 아닌 경우 에러 메시지 표시
+function printEmailErrorMessage() {
+  const value = inputEmail.value;
+  if (value === '') {
```

Comment:
아래처럼 조건문을 걸면 falsy한 값들에도 대응 할 수 있어서 더 좋을거 같네요!

```suggestion
  if (!value) {
```